components/vf-sass-config/variables/vf-variables.scss

```diff
@@ -1,19 +1,19 @@
 // Rollup of global Sass variables.
 /* stylelint-disable */
-@import 'vf-design-tokens/dist/sass/vf-border-radius.variables.scss';
-@import 'vf-design-tokens/dist/sass/vf-breakpoints.variables.scss';
-@import 'vf-design-tokens/dist/sass/maps/vf-breakpoints.map.scss';
-@import 'vf-design-tokens/dist/sass/maps/vf-colors.map.scss';
-@import 'vf-design-tokens/dist/sass/maps/vf-ui-colors.map.scss';
-@import 'vf-design-tokens/dist/sass/vf-font--monospace.scss';
-@import 'vf-design-tokens/dist/sass/vf-font--sans.scss';
-@import 'vf-design-tokens/dist/sass/vf-font-families.variables.scss';
-@import 'vf-design-tokens/dist/sass/vf-links.variables.scss';
-@import 'vf-design-tokens/dist/sass/maps/vf-spacing.map.scss';
-@import 'vf-design-tokens/dist/sass/maps/vf-zindex.map.scss';
+@import '../../vf-design-tokens/dist/sass/vf-border-radius.variables.scss';
```

Comment:
Checked this with a local project using a customised `compontents/vf-design-tokens` and it still works as expected.